environment.yml

```diff
@@ -23,11 +23,12 @@ dependencies:
     - scipy>=1.2
     - xarray>=2022.06.0
     # Extras
-    - wheel
+    - clisops
     - eofs
     - eigen
+    - flox
     - pybind11
```

Comment:
`eigen` and `pybind11` could be removed if we don't install SBCK with this env ?

Reply:
That's fair. Good idea!